spec/serializers/move_serializer_spec.rb

```diff
@@ -65,7 +65,7 @@
     let(:expected_json) do
       [
         {
-          id: move.person_id,
+          id: move.profile.person.id,
```

Comment:
is this definitely the `person_id` and not the `profile_id` ?

Reply:
Yes - the API exposes the Person object. Profile is currently hidden by the API.